packages/chats/src/Lightbox/Duplicate.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import type { DuplicateProps } from './types';
+import { DuplicateContent } from './DuplicateContent';
+
+export const Duplicate = (props: DuplicateProps) => {
+    const { isOpen } = props;
+
+    if (!isOpen) {
+        return null;
+    }
+
+    return (
+        <DuplicateContent
+            onClose={props.onClose}
+            originalRef={props.originalRef}
```

Comment:
What do you think to call it `forwardedRef`? It seems to be more widely used in React community :)